packages/showcase/pages/gmail.tsx

```diff
@@ -2,7 +2,7 @@ import clsx from "clsx";
 import { CursorProps, NodeApi, NodeRendererProps, Tree } from "react-arborist";
 import { gmailData, GmailItem } from "../data/gmail";
 import * as icons from "react-icons/md";
-import styles from "../styles/gmail.module.css";
+import styles from "../styles/Gmail.module.css";
```

Comment:
I use Linux and my filesystem (like most Linux filesystems) is case-sensitive so it needs to match the filename.